new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItCrossDomainTransaction.java

```diff
@@ -0,0 +1,400 @@
+
+    //build application archive
+    //Path application = Paths.get(RESOURCE_DIR, "apps", "txpropagate");
+    Path application = Paths.get(RESOURCE_DIR, "apps", "txforward");
+    BuildApplication.buildApplication(application, null, "build", domain1Namespace);
+
+    // create admin credential secret for domain1
+    logger.info("Create admin credential secret for domain1");
+    String domain1AdminSecretName = domainUid1 + "-weblogic-credentials";
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(
+        domain1AdminSecretName, domain1Namespace, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT),
+        String.format("createSecret %s failed for %s", domain1AdminSecretName, domainUid1));
+
+    // create admin credential secret for domain2
+    logger.info("Create admin credential secret for domain2");
+    String domain2AdminSecretName = domainUid2 + "-weblogic-credentials";
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(
+        domain2AdminSecretName, domain2Namespace, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT),
+        String.format("createSecret %s failed for %s", domain2AdminSecretName, domainUid2));
+
+    //createImageVerify expects the location of the ear file
+    String appSource = PV_ROOT + "/applications/" + WDT_APP_NAME + "/" + WDT_APP_NAME + ".ear";
+
+    logger.info("Creating image with model file and verify");
+    String domain1Image = createImageAndVerify(
+        WDT_IMAGE_NAME1, WDT_MODEL_FILE_DOMAIN1, appSource, WDT_MODEL_DOMAIN1_PROPS, PROPS_TEMP_DIR, domainUid1);
+
+    // docker login and push image to docker registry if necessary
+    dockerLoginAndPushImageToRegistry(domain1Image);
+
+    logger.info("Creating image with model file and verify");
+    //String domain2Image = createImageAndVerify(
+    //    WDT_IMAGE_NAME2, WDT_MODEL_FILE_DOMAIN2, WDT_APP_NAME, WDT_MODEL_DOMAIN2_PROPS, PROPS_TEMP_DIR, domainUid2);
+    String domain2Image = createImageAndVerify(
+        WDT_IMAGE_NAME2, WDT_MODEL_FILE_DOMAIN2, appSource, WDT_MODEL_DOMAIN2_PROPS, PROPS_TEMP_DIR, domainUid2);
+
+    // docker login and push image to docker registry if necessary
+    dockerLoginAndPushImageToRegistry(domain2Image);
+
+    //create domain1
+    createDomain(domainUid1, domain1Namespace, domain1AdminSecretName, domain1Image);
+    //create domain2
+    createDomain(domainUid2, domain2Namespace, domain2AdminSecretName, domain2Image);
+
+    logger.info("Getting admin server external service node port");
+    int adminServiceNodePort = assertDoesNotThrow(
+        () -> getServiceNodePort(domain1Namespace, domain1AdminServerPodName + "-external", "default"),
+        "Getting admin server node port failed");
+
+    String curlRequest = String.format("curl -v --show-error --noproxy '*' "
+            + "http://%s:%s/TxForward/TxForward?urls=t3://%s.%s:7001,t3://%s1.%s:8001,t3://%s1.%s:8001",
+             K8S_NODEPORT_HOST, adminServiceNodePort, domain1AdminServerPodName, domain1Namespace,
```

Comment:
add domain2-managed-server2 to end of URL list

Reply:
added domain2-managed-server2 to the url list